server/routes/shared/action-plan/actionPlanView.ts

```diff
@@ -39,6 +39,17 @@ export default class ActionPlanView {
     }
   }
 
+  get approvedActionPlansTableArgs(): TableArgs | null {
+    if (!this.presenter.showActionPlanVersions) {
```

Comment:
I personally don't think it is necessary to have this additional hide logic since it's the `njk` responsibility on weather to display(/call) the table or not

Reply:
Have removed this